### ISSUE TITLE
Internal error: Expected instance of PhpParser\Node\Name for $node->name, PhpParser\Node\Expr\ArrayDimFetch given

### ISSUE THREAD

**Issue by jdeniau** [ekino/phpstan-banned-code#13]
Hi, I got this weird error when trying to use your package.

The result of the debug is this:

```
Note: Using configuration file <path>/phpstan.neon.
<path>/src/Mapado/TicketingBundle/Export/ProTicketListGenerator.php
PHP Fatal error:  Uncaught RuntimeException: Expected instance of PhpParser\Node\Name for $node->name, PhpParser\Node\Expr\ArrayDimFetch given in <path>/vendor/ekino/phpstan-banned-code/src/Rules/BannedNodesRule.php:64
Stack trace:
#0 <path>/vendor/phpstan/phpstan/src/Analyser/Analyser.php(154): Ekino\PHPStanBannedCode\Rules\BannedNodesRule->processNode(Object(PhpParser\Node\Expr\FuncCall), Object(PHPStan\Analyser\Scope))
#1 <path>/vendor/phpstan/phpstan/src/Analyser/NodeScopeResolver.php(1691): PHPStan\Analyser\Analyser->PHPStan\Analyser\{closure}(Object(PhpParser\Node\Expr\FuncCall), Object(PHPStan\Analyser\Scope))
#2 <path>/vendor/phpstan/phpstan/src/Analyser/NodeScopeResolver.php(1139): PHPStan\Analyser\NodeScopeResolver->callNodeCallbackWithExpression(Object(Closure), Object(PhpParser\Node\Expr\FuncCall), Object(PHPStan\Analyser\Scope), Object(PHPStan\Analyser\ExpressionContext))
#3 <path>/vendor/phpstan/phpstan/s in <path>/vendor/ekino/phpstan-banned-code/src/Rules/BannedNodesRule.php on line 64
```

I do not know where it comes from, I try to parse a 1500 lines file :/

**Comment by mremi** [comment]
Hi @jdeniau ,

Can you show us `ProTicketListGenerator.php`?

**Comment by jdeniau** [comment]
Apparently, the bug comes from this line:

```
$cellValue = is_callable($column['value']) ? $column['value']($ticket) : $column['value'];
```

and more particularly the `$column['value']($ticket)` call

**Comment by mremi** [comment]
Workaround until it will be fixed:
```php
$cellValue = is_callable($column['value']) ? call_user_func($column['value'], $ticket) : $column['value'];
```

or assign `$column['value']` to an intermediate variable.

**Comment by nanofelis** [comment]
Fix #18
